lib/worldwide/config.rb

```diff
@@ -96,18 +100,36 @@ def expanded_locales_from_configuration(i18n_config)
       end
 
       def add_cldr_data(i18n_config, additional_components:)
+        locale_set = i18n_config.available_locales.map(&:to_s).to_set
         components = REQUIRED_CLDR_DATA + additional_components
-        locales = i18n_config.available_locales
-        i18n_config.load_path +=
-          Dir[File.join(Paths::CLDR_ROOT, "locales", "{#{locales.join(",")}}/{#{components.join(",")}}.{yml,rb}")]
```

Comment:
`{#{locales.join(",")}}` and `{#{components.join(",")}}` kill performance. We replace those with simple `"*"` wildcards.